runtime/doc/usr_01.txt

```diff
@@ -70,52 +70,18 @@ If the file already exists you probably want to keep it.
 For more info see |vimrc|.
 
 ==============================================================================
-*01.3*	Using the Vim tutor				*tutor* *vimtutor*
+*01.3*	Using the Vim tutor			      	            *tutor*
```

Comment:
There is no longer anything called "vimtutor", but I can restore it for compatibiility if you think it's necessary.